tests/tests/test-class-admin.php

```diff
@@ -475,4 +529,24 @@ private function dummy_meta_data( $stream_id ) {
 			'meta_value' => 'false',
 		);
 	}
+
+	/**
+	 * Login with a specific role.
+	 *
+	 * @param  string $role Use role.
+	 * @return int User ID.
+	 */
+	private function login_with_role( $role ) {
+		$user_id = $this->factory->user->create(
+			array(
+				'role'       => $role,
+				'user_login' => 'test_admin',
+				'email'      => 'test@land.com',
```

Comment:
In case one tests uses multiple roles, you don't want it to run into errors for non-unique emails and such.
```suggestion
				'user_login' => "test_{$role}}",
				'email'      => "test_{$role}@land.com",
```